source/wfs/text/operations.rst

```diff
@@ -358,7 +358,7 @@ The response returned by the request above is shown below. The response is prese
   </wfs:member>
   </wfs:FeatureCollection>
 
-PropetyName parameter
+PropertyName parameter
 ^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This line.

Reply:
Thank you! Will fix it and create a new PR, my bad!